docs/src/test-api/class-testinfo.md

```diff
@@ -31,6 +31,7 @@ Learn more about [test annotations](../test-annotations.md).
   - `contentType` <[string]> Content type of this attachment to properly present in the report, for example `'application/json'` or `'image/png'`.
   - `path` ?<[string]> Optional path on the filesystem to the attached file.
   - `body` ?<[Buffer]> Optional attachment body used instead of a file.
+  - `targetPath` ?<[string]> Path on the filesystem to the original file in project to which this attachment relates to.
```

Comment:
We treat public API as a huge maintenance cost; all changes or additions to the surface have to be justified and approved by all core team members.

In practice, only core functionality and highest pressing concerns result in public API changes. The "Accept screenshot button" is not important enough to justify this change, so this can't go in.